src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -47,13 +47,22 @@ angular
     '$$rAF',
     '$window',
     '$mdProgressCircular',
+    '$mdTheming',
     '$mdUtil',
     '$interval',
     '$log',
     MdProgressCircularDirective
   ]);
 
-function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdUtil, $interval, $log) {
+function MdProgressCircularDirective(
+  $$rAF,
+  $window,
+  $mdProgressCircular,
+  $mdTheming,
+  $mdUtil,
+  $interval,
+  $log
```

Comment:
What about this parameter line breaking? I've never seen that before. I suggest the following [code style](https://github.com/angular/material/blob/master/src/components/autocomplete/js/autocompleteController.js#L10)